### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by retroflex**
First git use ever, so might be some issues... :)

Added separate date format option to calendar for full day events.
As I use time in my calendar events, it listed full day events with time as well (midnight I think). Looks much cleaner when you can drop time for full day events and only show date for them.

### CODE REVIEW

**Comment by fewieden** [comment]
```
/home/travis/build/MichMich/MagicMirror/modules/default/calendar/calendar.js

line 28:5  error  Expected indentation of 2 tabs but found 4 spaces  indent
```

File: modules/default/calendar/calendar.js [hunk 1]
```diff
@@ -228,7 +229,7 @@ Module.register("calendar", {
 							// This event falls within the config.urgency period that the user has set
 							timeWrapper.innerHTML = this.capFirst(moment(event.startDate, "x").fromNow());
 						} else {
-							timeWrapper.innerHTML = this.capFirst(moment(event.startDate, "x").format(this.config.dateFormat));
+							timeWrapper.innerHTML = this.capFirst(moment(event.startDate, "x").format(this.config.fullDayEventDateFormat));
```
Comment (roramirez) [inline modules/default/calendar/calendar.js]: What difference is between dateFormat?. I can't undertand difference for add other configuration option for this.

Comment (fewieden) [inline modules/default/calendar/calendar.js]: You dont want time to be displayed on a fullday event. So you can set different formats for different event types.

Comment (retroflex) [inline modules/default/calendar/calendar.js]: Example of current calendar (left) and after my change (right):

mirrorcal

**Pull request merged**